src/libstd/io/mod.rs

```diff
@@ -1270,10 +1270,6 @@ pub trait BufRead: Read {
     /// delimiter or EOF is found. Once found, all bytes up to, and including,
     /// the delimiter (if found) will be appended to `buf`.
     ///
-    /// If this reader is currently at EOF then this function will not modify
-    /// `buf` and will return `Ok(n)` where `n` is the number of bytes which
-    /// were read.
```

Comment:
Added some actual description of successful return value, and done similar
change to read_line.